docs/inference.md

```diff
@@ -79,6 +79,14 @@ To run the code infilling example:
 python examples/code_llama/code_infilling_example.py --model_name MODEL_NAME --prompt_file examples/code_llama/code_infilling_prompt.txt --temperature 0.2 --top_p 0.9
 
 ```
+To run the 70B Instruct model example run the following, it asks for system and user prompt to instruct the model:
```

Comment:
=> To run the 70B Instruct model example (you'll need to enter the system and user prompts to instruct the model):